open_fdd/tests/desktop/test_desktop_bridge.py

```
@@ -469,6 +469,283 @@ def test_data_model_health_summary_endpoint_shape() -> None:
+            json={"profiles_yaml": str(yml.resolve()), "reset": True},
+        )
+        assert res.status_code == 200, res.text
+        body = res.json()
+        assert body.get("sites")
+
+
+def test_plots_fdd_frame_returns_fault_columns(monkeypatch: pytest.MonkeyPatch, tmp_path: Path) -> None:
+    pytest.importorskip("pyarrow")
+    monkeypatch.setenv("OFDD_DESKTOP_DATA_DIR", str(tmp_path / "fdd_demo"))
+    repo_root = Path(__file__).resolve().parents[3]
+    rules_pack = default_rules_root() / "ahu_vav"
```

Ruff (0.15.12)

[error] Undefined name `default_rules_root`

(F821)